build/README.md

```diff
@@ -6,7 +6,7 @@ Docker Script for building `pri-fidoiot` repository. Using this script you can b
 
 - Operating system: **Ubuntu 20.04.**
 
-- Docker engine : **18.06.0**
+- Docker engine : **18.06.0** (Supported till version <=20.10.7)
```

Comment:
Consider using same note in both places, can remove the <= symbol.

Reply:
Sure.